test/test-helper.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+global.chai = require('chai');
+global.expect = chai.expect;
+global.sinon = require('sinon');
+
+chai.use(require('sinon-chai'));
```

Comment:
will sinon-chai working globally like this? I usually have to do
```
global.chai = require('chai')
  .use(require('sinon-chai'))
```

Reply:
It's working globally for me!